src/effects/EffectUI.cpp

```diff
@@ -309,15 +309,24 @@ int EffectUIHost::ShowModal()
    mApplyBtn->Destroy();
    mApplyBtn = apply;
    mApplyBtn->SetDefault();
-   mApplyBtn->SetLabel(wxGetStockLabel(wxID_OK, 0));
+
+   if (mIsBatch)
```

Comment:
Probably a wrong condition?

Reply:
Changed to use mpTempProjectState instead, plus tacked on a few other condition checks as well.